middleware/2fa_service/messageContent.js

```diff
@@ -0,0 +1,139 @@
+const hex = require('hexer');
+const nearAPI = require('near-api-js');
+const escapeHtml = require('escape-html');
+
+const fmtNear = (amount) => nearAPI.utils.format.formatNearAmount(amount, 4) + 'Ⓝ';
+
+const TRUNCATE_ARGUMENTS_LENGTH = 247;
+
+const formatArgs = (args, isForSms) => {
+    let output = '';
+
+    const argsBuffer = Buffer.from(args, 'base64');
+    try {
+        const jsonString = argsBuffer.toString('utf-8');
+        const parsed = JSON.parse(jsonString);
+
+        // Composing new obj in specific order so `amount` and `deposit` will likely be in the first 250chars
```

Comment:
this seems to be some new logic, what is it's purpose?

if this is trying to format NEAR amounts in method call args, we shouldn't do it as args can be anything we cannot be sure that `amount` there is actual NEAR tokens and so can misrepresent what is going on